configuration/configuration.go

```diff
@@ -12,6 +12,7 @@ import (
 	"github.com/containous/traefik/log"
 	"github.com/containous/traefik/middlewares/tracing"
 	"github.com/containous/traefik/ping"
+	acmeprovider "github.com/containous/traefik/provider/acme"
```

Comment:
You used `acmeprovider` but [here](https://github.com/containous/traefik/pull/2889/files#diff-bb76aa29d7113847a8b689dc0e0b95ccR12) you used `providerACME`.
Could you rename one to be consistent please ?

Reply:
Every instances renamed into `acmeprovider`, sir!